Project.toml

```diff
@@ -3,15 +3,17 @@ uuid = "1313f7d8-7da2-5740-9ea0-a2ca25f37964"
 version = "0.7.1"
 
 [deps]
+Chain = "8be319e6-bccf-4806-a6f7-6fae938471bc"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 
 [compat]
 DataFrames = "0.21, 0.22, 1"
+MacroTools = "0.5"
 Reexport = "0.2, 1"
 julia = "1"
-MacroTools = "0.5"
+Chain = "0.4"
```

Comment:
do we have CompatHelper installed for this repo?

Reply:
Yeah I will work on that.